File: java/json/samples/producer/src/main/java/com/azure/schemaregistry/samples/CustomerInvoice.java
package com.azure.schemaregistry.samples;

public class CustomerInvoice {
  private String invoiceId;
  private String merchantId;
  private int transactionValueUsd;
  private String userId;

  public CustomerInvoice() {}

  public CustomerInvoice(String invoiceId, String merchantId, int transactionValueUsd, String userId) {
    this.invoiceId = invoiceId;
    this.merchantId = merchantId;
    this.transactionValueUsd = transactionValueUsd;
    this.userId = userId;
  }

  public String getInvoiceId() {
    return this.invoiceId;
  }

  public void setInvoiceId(String id) {
    this.invoiceId = id;
  }

  public String getMerchantId() {
    return this.merchantId;
  }

  public void setMerchantId(String id) {
    this.merchantId = id;
  }

  public int getTransactionValueUsd() {
    return this.transactionValueUsd;
  }

  public void setTransactionValueUsd(int transactionValueUsd) {
    this.transactionValueUsd = transactionValueUsd;
  }

  public String getUserId() {
    return this.userId;
  }

  public void setUserId(String userId) {
    this.userId = userId;
  }
}

File: csharp/json/samples/CustomerInvoice.cs
//Copyright (c) Microsoft Corporation. All rights reserved.
//Copyright 2016-2017 Confluent Inc., 2015-2016 Andreas Heider
//Licensed under the MIT License.
//Licensed under the Apache License, Version 2.0
//
//Original Confluent sample modified for use with Azure Event Hubs for Apache Kafka Ecosystems

namespace com.azure.schemaregistry.samples
{
    using System;
    using Newtonsoft.Json;

    [JsonObject]
    public class CustomerInvoice
    {
        [JsonProperty]
        public string InvoiceId { get; set; }
        
        [JsonProperty]
        public string MerchantId { get; set; }
        
        [JsonProperty]
        public int TransactionValueUsd { get; set; }

        [JsonProperty]
        public string UserId { get; set; }
    }
}
